terraform/environments/core-vpc/iam.tf

```diff
@@ -1,3 +1,48 @@
 data "aws_iam_role" "vpc-flow-log" {
   name = "AWSVPCFlowLog"
 }
+
+resource "aws_iam_policy" "policy" {
+  name        = "s3_dynamo_access_policy"
+  path        = "/"
+  description = "S3 and dynamo policy"
+
+  policy = jsonencode({
+    "Version" : "2012-10-17",
+    "Statement" : [
+      {
+        "Effect" : "Allow",
+        "Action" : "s3:ListBucket",
+        "Resource" : "arn:aws:s3:::mybucket"
+      },
+      {
+        "Effect" : "Allow",
+        "Action" : ["s3:GetObject", "s3:PutObject", "s3:DeleteObject"],
+        "Resource" : "arn:aws:s3:::modernisation-platform-terraform-state"
+      }
+    ]
+    "Version" : "2012-10-17",
+    "Statement" : [
+      {
+        "Effect" : "Allow",
+        "Action" : [
+          "dynamodb:DeleteItem",
+          "dynamodb:GetItem",
+          "dynamodb:PutItem",
+          "dynamodb:Query",
+          "dynamodb:UpdateItem"
+        ],
+        "Resource" : ["arn:aws:dynamodb:eu-west-2:${local.environment_management.modernisation_platform_account_id}:table/modernisation-platform-terraform-state-lock"],
+        "Condition" : {
+          "ForAllValues:StringEquals" : {
+            "dynamodb:LeadingKeys" : [
+              "myorg-terraform-states/myapp/production/tfstate",    // during a state lock the full state file is stored with this key
```

Comment:
Are these key values correct? I suspect we're not using `myorg-terraform-states/myapp` in the DynamoDB config.

Reply:
May be corrected with "modernisation-platform-terraform-state/global-resources/terraform.tfstate" and "modernisation-platform-terraform-state/global-resources/terraform.tfstate-md5"

Reply:
Change made and pushed